spec/models/task_spec.rb

```diff
@@ -23,30 +24,40 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  context 'factory tests' do
-    subject { build(:task) }
-    it { is_expected.to be_valid }
-  end
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user_id: user.id) }
+  let!(:project) { create(:project, workspace_id: workspace.id) }
+  let!(:task1) { create(:task, project_id: project.id) }
+  let!(:task2) { create(:task, project_id: project.id) }
+  let!(:task3) { create(:task, project_id: project.id) }
+
+  context 'test before_save :section' do
+    let!(:task5) { create(:task, project_id: project.id) }
+    let!(:task6) { create(:task, title: 'Section:',  project_id: project.id) }
+    let!(:task7) { create(:task, title: 'Test : text') }
+
+    it 'should set status section as false' do
+      expect(task5.section).to eq false
```

Comment:
nitpick:
* `expect(task5).not_to be_section`
* `expect(task5.section?).to be_falsey`
* `expect(task5.section).to be_falsey`